src/frontend/youtube_clone/src/app/app.module.ts

```diff
@@ -1,16 +1,36 @@
 import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
-
+import { MatSlideToggleModule } from '@angular/material/slide-toggle';
 import { AppRoutingModule } from './app-routing.module';
+
 import { AppComponent } from './app.component';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
```

Comment:
Try to group your imports. I don't like that you mix @angular with your own components

Reply:
regrouped them, by source and by use